### PR TITLE
Update maintenance to 2.4.7

### PR DESCRIPTION
**Pull request by ran-dall**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] The commit message includes the cask’s name and version.

### CODE REVIEW

File: Casks/maintenance.rb [hunk 1]
```diff
@@ -23,7 +23,7 @@
     version '2.3.0'
     sha256 '8fde91742126d10234451a3c973461f5d84c771e52c6ee14aff93f1d66a0dbca'
   elsif MacOS.version == :high_sierra
```
Comment (reitermarkus) [inline Casks/maintenance.rb]: If `:high_sierra` and `else` are the same, just remove this.

Comment (ran-dall) [reply Casks/maintenance.rb]: @reitermarkus Good catch! I updated it accordingly. Thank you!

Comment (ran-dall) [reply Casks/maintenance.rb]: @reitermarkus Actually, they are slightly different. Re-updated.

**Comment by vitorgalvao** [comment]
With the latest commit, this is changing nothing.

**Comment by ran-dall** [comment]
@vitorgalvao This one was kinda weird. Each version has a different version and sha that's listed on their website (https://www.titanium-software.fr/en/maintenance.html). I downloaded side by side to confirm the version for each. That's why you see so many commits. Here are some screens...

screen shot 2018-11-19 at 12 11 55 pm
screen shot 2018-11-19 at 12 14 36 pm

**Comment by vitorgalvao** [comment]
@ran-dall But [there is nothing being changed](https://github.com/Homebrew/homebrew-cask/pull/55262).

**Pull request closed without merging**